devel/use_case.md

```diff
@@ -0,0 +1,41 @@
+<!-- vim-markdown-toc GFM -->
+
+* [Set static IP address](#set-static-ip-address)
+
+<!-- vim-markdown-toc -->
+Python code examples for common use cases.
+
+# Set static IP address
+
```

Comment:
Can we add the use case "story" here?
Something like:
`Define an ethernet interface with two static IPv4 addresses and two static IPv6 addresses`
Note: The DHCP4/6 and AUTOCONF are optional.